### PR TITLE
fixed font-size and add truncate for long titel & description

### PR DESCRIPTION
**Pull request by WahidinAji**
i try to make a new blog last week, but I forgot to finish that. and now I try to fixed font-size on the title and others.

### CODE REVIEW

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+telegram:
+cover: image/pinguin.png
+date: 2022-01-05
+categories:
+  - laravel
+  - tips
+  - ubuntu
+---
+
+setelah sempat pensiun jadi pengguna penguin (Linux), akhirnya dari sekitar dua minggu yang lalu saya kembali memakai full linux untuk proses project development saya. Nah, maka dari itu untuk kali ini saya akan mencoba membagikan beberapa tips untuk setup Linux bagi para Laravel Developer. Oiya untuk distro Linux yang pakai adalah Ubuntu Linux versi 18.04 LTS.
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:16]:
```suggestion
Setelah sempat pensiun jadi pengguna penguin (Linux), akhirnya dari sekitar dua minggu yang lalu saya kembali memakai full linux untuk proses project development saya. Nah, maka dari itu untuk kali ini saya akan mencoba membagikan beberapa tips untuk setup Linux bagi para Laravel Developer. Oiya untuk distro Linux yang pakai adalah Ubuntu Linux versi 18.04 LTS.
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+setelah proses diatas sudah dijalankan satu-persatu, kemudian pindahkan composer.phar ke direktori PATH.
+
+```bash
+sudo mv composer.phar /usr/local/bin/composer
+```
+
+# NodeJs
+
+Kita perlu memasang NodeJS untuk memasang NPM (Node Package manager), caranya pun cukup mudah. Jalankan perintah dibawah ini satu per satu,
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:47]:
```suggestion
Kita perlu memasang NodeJS untuk memasang NPM (Node Package manager), caranya pun cukup mudah. Jalankan perintah dibawah ini satu per satu:
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+Kemudian pasang composer,
+
+```bash
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+php -r "if (hash_file('sha384', 'composer-setup.php') === '906a84df04cea2aa72f40b5f787e49f22d4c2f19492ac310e8cba5b96ac8b64115ac402c8cd292b8a03482574915d1a8') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
+php composer-setup.php
+php -r "unlink('composer-setup.php');"
+```
+
+setelah proses diatas sudah dijalankan satu-persatu, kemudian pindahkan composer.phar ke direktori PATH.
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:39]:
```suggestion
Setelah proses diatas sudah dijalankan satu-persatu, kemudian pindahkan composer.phar ke direktori PATH.
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+Kita perlu memasang NodeJS untuk memasang NPM (Node Package manager), caranya pun cukup mudah. Jalankan perintah dibawah ini satu per satu,
+
+```bash
+curl -sL https://deb.nodesource.com/setup_17.x -o nodesource_setup.sh
+sudo bash nodesource_setup.sh
+sudo apt install nodejs
+```
+
+untuk version pada `https://deb.nodesource.com/setup_17.x`, ini opsional ya. kalian bisa menyesuaikan, sesuai dengan kebutuhan.
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:55]:
```suggestion
Untuk version pada `https://deb.nodesource.com/setup_17.x`, ini opsional ya. kalian bisa menyesuaikan, sesuai dengan kebutuhan.
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+# MySQL
+
+Nah, untuk mysql ini kalian sebenarnya tidak perlu bingung. Cukup buka link ini
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:59]:
```suggestion
Nah, untuk MySQL ini kalian sebenarnya tidak perlu bingung. Cukup buka link ini
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+- [`Mysql Download`](https://dev.mysql.com/downloads/mysql/),
+
+download mysqlnya dan sesuaikan dengan OS Distro Linux kalian. Kemudian ikuti langkah-langkahnya seperti yang ada di dokumentasi ini
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:63]:
```suggestion
Download MySQL-nya dan sesuaikan dengan OS Distro Linux kalian. Kemudian ikuti langkah-langkahnya seperti yang ada di dokumentasi ini
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+- [`Cara Install`](https://dev.mysql.com/doc/refman/8.0/en/linux-installation-debian.html).
+
+lohh, kok ke dokumentasi? ya sekalian belajar baca-baca dokumentasinya juga hehehe :). Belajar dari yang mudah untuk diikuti terlebih dahulu aja okay ^~^.
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:67]:
```suggestion
Lohh, kok ke dokumentasi? Ya sekalian belajar baca-baca dokumentasinya juga hehehe :). Belajar dari yang mudah untuk diikuti terlebih dahulu aja okay ^~^.
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+Untuk MySql Client nya sendiri, saya menggunakan ekstensi Database yang ada di Visual Studio Code. Jadi, saya ga perlu install phpmyadmin atau MySql Client sejenisnya.
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:69]:
```suggestion
Untuk MySQL Client nya sendiri, saya menggunakan ekstensi Database yang ada di Visual Studio Code. Jadi, saya ga perlu install PHPMyAdmin atau MySQL Client sejenisnya.
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+Oiya, disini saya tidak menggunakan Web Server ya. karena Laravel sendiri sudah menyediakan dengan cukup jalankan command `php artisan serve`.
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:71]:
```suggestion
Oiya, disini saya tidak menggunakan Web Server ya. Karena Laravel sendiri sudah menyediakan dengan cukup jalankan command `php artisan serve`.
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+# Go
+
+Sebenarnya, langkahnya cukup mudah. Pertama [Download Go disini](https://go.dev/doc/install), setelah itu jalankan perintah dibawah ini satu per satu
+
+```bash
+rm -rf /usr/local/go && tar -C /usr/local -xzf go1.17.6.linux-amd64.tar.gz
+export PATH=$PATH:/usr/local/go/bin
+```
+
+udah deh, tinggal cek. Apakah sudah terpasang atau belum dengan command
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:84]:
```suggestion
Udah deh, tinggal cek. Apakah sudah terpasang atau belum dengan command
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+
+```bash
+go version
+```
+
+# PosgreSql
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:90]:
```suggestion
# PostgreSQL
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+
+Yapss, karena kebanyakan Go Developer lebih suka menggunakan PostgreSql dari pada MySql. Maka kita akan install juga, caranya ini cukup mudah. Cukup ikuti langkah-langkahnya seperti yang ada di dokumentasi resmi ini
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:92]:
```suggestion
Yapss, karena kebanyakan Go Developer lebih suka menggunakan PostgreSQL dari pada MySQL. Maka kita akan install juga, caranya ini cukup mudah. Cukup ikuti langkah-langkahnya seperti yang ada di dokumentasi resmi ini
```

kata siapa Go developer lebih suka PostgreSQL dibanding MySQL?

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 13]
```diff
@@ -0,0 +1,111 @@
...
+
+- [Cara Install PostgreSql](https://www.postgresql.org/download/linux/ubuntu/)
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:94]:
```suggestion
- [Cara Install PostgreSQL](https://www.postgresql.org/download/linux/ubuntu/)
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+
+Kalau sudah, untuk PostgreSql Clientnya sendiri, saya juga menggunakan ekstensi yang ada di Visual Studio Code / Goland.
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:96]:
```suggestion
Kalau sudah, untuk PostgreSQL Clientnya sendiri, saya juga menggunakan ekstensi yang ada di Visual Studio Code / Goland.
```

File: content/setup-ubuntu-linux-for-laravel-developer.md [hunk 15]
```diff
@@ -0,0 +1,111 @@
...
+
+`kalian : boleh minta link ekstensinya ga om?`
+
+`saya : boleh, tapi enggak dibahas disini ya, 😁😁😁.`
+
+kenapa saya beberapa langkah saya arahkan ke dokumentasi langsung? ya sambil belajar baca dokumentasi dikit-dikitlah ya, supaya enggak terjebak di tutorial hell mhehehe :)
```
Comment (aldy505) [inline content/setup-ubuntu-linux-for-laravel-developer.md:102]:
```suggestion
Kenapa saya beberapa langkah saya arahkan ke dokumentasi langsung? Ya sambil belajar baca dokumentasi dikit-dikitlah ya, supaya enggak terjebak di tutorial hell mhehehe :)
```

**Review by aldy505 — CHANGES_REQUESTED** [review]
typo

File: content/setup-ubuntu-linux-for-laravel-developer.md (hunk 1 above)
Comment (WahidinAji) [reply content/setup-ubuntu-linux-for-laravel-developer.md]: Ok, sementara tak delete dulu. biar engga ketumpuk.

**Review by aldy505 — APPROVED** [review]

**Pull request merged**